Untitled Pull Request

Ethan-mo opened the pull request:
기본 View 설정
    1. Home View
    2. Detail View
        1) 글 보기1
        2) 글 보기2
        3) 글 수정
    3. Setting View
        1) 개인정보 설정 창
        2) 로그인 옵션 창(SNS 계정 로그인)
        
📕Plan
1. 캘린더 View 추가
2. Home View Defalut UI 설정

Ethan-mo committed:
🎨 [style]
1. UI 제작 : Home, Detail, Setting View 구현

jkim68888 commented on DiaryApp/DiaryApp/ViewController.swift:
```diff
@@ -9,11 +9,33 @@ import UIKit
+		homeViewSetting()
 	}
+    func homeViewSetting(){
+        homeInfoView.backgroundColor = #colorLiteral(red: 0.9098039269, green: 0.4784313738, blue: 0.6431372762, alpha: 1)
```
color값은 UIColor를 extension해서 hex값으로 쓰면 편해용~

jkim68888 commented on DiaryApp/DiaryApp/ViewController.swift:
```diff
@@ -9,11 +9,33 @@ import UIKit
+    override func viewDidLoad() {
 		super.viewDidLoad()
-		// Do any additional setup after loading the view.
+		homeViewSetting()
```
HomeView 관련 ui를 여기서 세팅하지 말고 HomeViewController 에서 세팅해야될듯!

jkim68888 commented on DiaryApp/DiaryApp/ViewController.swift:
```diff
@@ -9,11 +9,33 @@ import UIKit
 
 class ViewController: UIViewController {
```
우리 스토리보드도 지금처럼 Main에 다 하면 안되구 파일 여러개로 나눠야 될거같앙!

Ethan-mo committed:
🎨 [style]
1. UI 제작 : Home, Detail storyboard 분할 구현

Ethan-mo committed:
🎨 [style]
1. UI 제작 : Home, Detail, Profile storyboard 분할 구현
2. VC제작 : Home, Detail, Profile .swift파일 분할 구현
3. MVC패턴으로 폴더링

jkim68888 committed:
conflict solved

jkim68888 committed:
Merge branch 'main' into Ethan
